### ISSUE TITLE
[BUG] Object navigation in Narrator not in the logical/ visual sequence

### ISSUE THREAD

**Issue by adarshrema** [microsoft/accessibility-insights-windows#700]
Issue: While using object navigation in Narrator, the disabled Save and close button is not announced in the logical/ visual navigation sequence
User impact: Screen reader users are presented with a navigation order that is different from logical/ visual order

Screen: Settings > Application

Repro steps:
1. Open Accessibility Insights for Windows and navigate to Settings (left nav panel)
2. Narrator announces the Application tab.
3. Use Caps key and left arrow to navigate to the object before Application tab
4. Narrator announces 'Disabled Save and close button'
5. Visually, the disabled Save and close button appears after all the settings options in the Applications tab
Note: This is a problem isolated to object navigation while settings are not updated. If any of the settings are changed, the active Save and close button focus is in the logical/ visual order while using the tab (or shift + tab) key

Expected behavior:
Screen readers are presented with the same navigation order as the logical/ visual order for the disabled Save and close button

**Comment by adarshrema** [comment]
validated in Canary
